Makefile

```diff
@@ -24,23 +24,23 @@ HUB ?=
 TAG ?=
 
 build:
-	@bazel $(BAZEL_STARTUP_ARGS) build $(BAZEL_BUILD_ARGS) //...
+	@${BAZEL_BIN} $(BAZEL_STARTUP_ARGS) build $(BAZEL_BUILD_ARGS) //...
 
 # Build only envoy - fast
 build_envoy:
-	bazel $(BAZEL_STARTUP_ARGS) build $(BAZEL_BUILD_ARGS) //src/envoy/mixer:envoy
+	${BAZEL_BIN} $(BAZEL_STARTUP_ARGS) build $(BAZEL_BUILD_ARGS) //src/envoy/tcp/mixer:filter_lib //src/envoy/http/mixer:filter_lib
```

Comment:
Why did you narrow the build target here?

Reply:
At the moment I proposed this change the target `//src/envoy/mixer:envoy` doesn't exist anymore. At that moment I assume it's better something than nothing.
I explored and I think the goal is to build an envoy binary. Will fix